agent/agent_installer/Windows/AgentService.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+<id>Hydra Lab Agent Service</id>
+<name>Hydra Lab Agent Service</name>
+<description>Hydra Lab Agent Service.</description>
```

Comment:
`. <description>Hydra Lab Test Agent Service acts as the test runner and test device manager of the Hydra Lab center.</description>`